packages/jsio.js

```diff
@@ -370,6 +363,17 @@
       this.eval = function(code, path) {};
       this.fetch = function(path) { return contentsOfPath; };
       this.log = function(args...) {};
+
+      this.getNamespace = function(key) { return CONFIG.shortName + ':' + key };
```

Comment:
Is there no way to identify a running jsio app?  Seems like there must be (I thought `CONFIG` was from jsio not devkit)